<launch>

<arg name="thename" default="m100"/><!--无人机名称（每台无人机必须不相同）-->
<arg name="axis" default="2"/><!--坐标系：1为全局自然坐标系，2为局部实验坐标系-->

<node
pkg = "delay"
type = "delayuav"
name = "delayuav_node"
output = "screen"
respawn = "true">
<param name="uav_name" type="string" value="$(arg thename)"/>
<param name="axis_id" type="int" value="$(arg axis)"/>
</node>

</launch>